### PR TITLE
DSP-13557: made lender search dropdown scroll with the user's selection

### PR DESCRIPTION
**Pull request by maxponte**
Made the container scroll to the user's selection

### CODE REVIEW

File: modules/directives/autocomplete/autocomplete.js [hunk 1]
```diff
@@ -142,60 +141,87 @@ angular.module('dataform.directives').directive('dfDatalist', [function() {
...
-          break;
-        case 13: // enter
-          if (typeof scope.activeIndex === 'number') {
-            selectItem(nthItem(scope.activeIndex), $event);
-          }
-          break;
-        case 27: // escape
-          elem.trigger('hide');
-          break;
+          case 40: // down arrow
```
Comment (ianlyons) [inline modules/directives/autocomplete/autocomplete.js]: ? why this diff?

File: modules/directives/autocomplete/autocomplete.js [hunk 2]
```diff
@@ -209,9 +235,13 @@ angular.module('dataform.directives').directive('dfDatalist', [function() {
         move($event);
       });
 
-      elem.delegate('li[df-value]', 'mouseenter', function($event) {
+      elem.delegate('li[df-value]', 'mousemove', function($event) {
```
Comment (ianlyons) [inline modules/directives/autocomplete/autocomplete.js]: ?

File: dist/angular-dataform.js [hunk 3]
```diff
@@ -70,9 +69,9 @@ angular.module('dataform.directives').directive('dfAutocompleteDatalist', ['$doc
 
       // Hide datalist when blurred, UNLESS we're hovering the datalist.
       // This is to avoid removing the datalist before the click event registers.
-      var $datalist_mousedOver = false;
-      $datalist.on('mouseenter', function() { $datalist_mousedOver = true; });
-      $datalist.on('mouseleave', function() { $datalist_mousedOver = false; });
+      var $datalist_mousedOver = scope.datalist_mousedOver = false;
```
Comment (ianlyons) [inline dist/angular-dataform.js]: why are you putting this on the cope?

Comment (maxponte) [inline dist/angular-dataform.js]: needs to be shared with child directive

File: modules/directives/autocomplete/autocomplete.js (hunk 1 above)
Comment (maxponte) [inline modules/directives/autocomplete/autocomplete.js]: just my editor, I'll revert it

**Pull request merged**